openassessment/templates/openassessmentblock/staff_area/oa_staff_override_assessment.html

```diff
@@ -39,7 +39,7 @@ <h3 class="message__title">{% trans "We could not submit your assessment" %}</h3
                 <ul class="list list--actions">
                     <li class="list--actions__item">
                         <button type="submit" class="action action--submit is--disabled">
-                            <span class="copy">{% trans "Submit your assessment" %}</span>
+                            <span class="copy">{% trans "Submit Assessment" %}</span>
```

Comment:
@efischer19 I asked Carol about this change in the main PR. I would remove this from your branch.